ninja-core/pom.xml

```diff
@@ -123,12 +123,6 @@
             <artifactId>freemarker-gae</artifactId>
         </dependency>
 
-        <!-- guava is just cool -->
-        <dependency>
-            <groupId>com.google.guava</groupId>
```

Comment:
This may break stuff for some users. On the other hand. Less libraries less problems. Removing guava is ok. We can re-add it if there are grave issues...